libgit2-sys/lib.rs

```diff
@@ -600,6 +603,7 @@ pub enum git_ref_t {
 }
 pub use git_ref_t::*;
 
+enum_from_primitive!{
```

Comment:
Yeah there's a few other enums that are exposed as `i32` as they're just a pile of flags. Or, to be more concrete, they're exposed as `__enum_ty` as apparently enums change signedness on MSVC!